packages/mc-scripts/config/create-webpack-config-for-development.js

```diff
@@ -56,7 +56,8 @@ module.exports = ({
     app: [
       // Ship a few polyfills by default
       require.resolve('./polyfills'),
-      require.resolve('babel-polyfill'),
+      require.resolve('core-js/stable'),
+      require.resolve('regenerator-runtime/runtime'),
```

Comment:
Replacing the deprecated `@babel/polyfill` requires to manually specifying those 2 polyfills:

https://babeljs.io/blog/2019/03/19/7.4.0#migration-from-core-js-2

Reply:
Correct.